### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by elishkaflint**
Added feature tests and controller tests

### CODE REVIEW

File: .idea/acebook-rails-keeping-it-rails.iml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="PROVIDED" name="diff-lcs (v1.3, RVM: ruby-2.5.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="rspec-core (v3.7.1, RVM: ruby-2.5.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="rspec-expectations (v3.7.0, RVM: ruby-2.5.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="rspec-mocks (v3.7.0, RVM: ruby-2.5.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="rspec-support (v3.7.1, RVM: ruby-2.5.1) [gem]" level="application" />
+  </component>
+</module>
```
Comment (neobay991) [inline .idea/acebook-rails-keeping-it-rails.iml]: Not sure what this is? please explain

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
+    <option name="languageLevel" value="ES6" />
+  </component>
+</project>
```
Comment (neobay991) [inline .idea/misc.xml]: Not sure what this is? please explain

File: .idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/acebook-rails-keeping-it-rails.iml" filepath="$PROJECT_DIR$/.idea/acebook-rails-keeping-it-rails.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (neobay991) [inline .idea/modules.xml]: Not sure what this is? please explain

File: .idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (neobay991) [inline .idea/vcs.xml]: Not sure what this is? please explain

**Review by neobay991 — CHANGES_REQUESTED** [review]

File: .idea/acebook-rails-keeping-it-rails.iml (hunk 1 above)
Comment (elishkaflint) [reply .idea/acebook-rails-keeping-it-rails.iml]: files auto-generated by daniel's jetbrain IDE, will remove as he is now using atom

File: .idea/modules.xml (hunk 3 above)
Comment (elishkaflint) [reply .idea/modules.xml]: files auto-generated by daniel's jetbrain IDE, will remove as he is now using atom

File: .idea/vcs.xml (hunk 4 above)
Comment (elishkaflint) [reply .idea/vcs.xml]: files auto-generated by daniel's jetbrain IDE, will remove as he is now using atom

File: .idea/misc.xml (hunk 2 above)
Comment (elishkaflint) [reply .idea/misc.xml]: files auto-generated by daniel's jetbrain IDE, will remove as he is now using atom

**Review by neobay991 — APPROVED** [review]
Awesome!

**Comment by neobay991** [comment]
:shipit:

**Pull request merged**